test/tests/customhighlights.js

```diff
@@ -96,4 +96,56 @@ describe("Custom highlights", function () {
 		client.compileCustomHighlights();
 		expect(client.highlightRegex).to.be.null;
 	});
+
+	// tests for highlight exceptions
+	client.config.clientSettings.highlights = "foo, @all,   sp ace   , 고";
+	client.config.clientSettings.highlightExceptions = "foo bar, bar @all, test sp ace test";
```

Comment:
You set these but don't call `compileCustomHighlights`?

Reply:
Oh, I forgot to add that, yes. The custom highlights are cleared in line 91 and 95, so I need to recompile anyway.